### DISCUSSION TITLE
zh/guide/drivers/139

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#40] [Announcements]
# zh/guide/drivers/139

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/139.html

**Comment by Carlos6668** [comment]
添加后提示个人云未知异常
failed get objs: failed to list objs: 个人云未知异常

**Reply by Saniq** [reply]
根文件id找一下我也遇到了

**Reply by ljx0021** [reply]
设置根目录去getdisk的response信息中去找catalogID就行啦，你想哪个做根目录都可以的。里面选一个就是了
{
    "success": true,
    "code": "0",
    "message": "OK",
    "data": {
        "result": {
            "resultCode": "0",
            "resultDesc": null
        },
        "getDiskResult": {
            "parentCatalogID": "root",
            "nodeCount": 7,
            "catalogList": [
                {
                    "catalogID": "1B11chpTJ14000019700101000000071",
                    "catalogName": "我的应用收藏",
                    "catalogType": 6,
                    "createTime": "20171229150259",
                    "updateTime": "20171229150259",
                    "isShared": false,
                    "catalogLevel": 2,
                    "shareDoneeCount": 0,
                    "openType": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "dirEtag": 4194304,
                    "tombstoned": 0,
                    "proxyID": null,
                    "moved": 0,
                    "isFixedDir": 1,
                    "isSynced": null,
                    "owner": "13910030347",
                    "modifier": "",
                    "path": "00019700101000000001/1B11chpTJ14000019700101000000071",
                    "shareType": 0,
                    "softLink": null,
                    "extProp1": null,
                    "extProp2": null,
                    "extProp3": null,
                    "extProp4": null,
                    "extProp5": null,
                    "ETagOprType": 1
                },
                {
                    "catalogID": "1B11chpTJ14000019700101000000043",
                    "catalogName": "手机图片",
                    "catalogType": 7,
                    "createTime": "20170329122757",
                    "updateTime": "20170329123212",
                    "isShared": false,
                    "catalogLevel": 2,
                    "shareDoneeCount": 0,
                    "openType": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "dirEtag": 4194304,
                    "tombstoned": 0,
                    "proxyID": null,
                    "moved": 0,
                    "isFixedDir": 1,
                    "isSynced": null,
                    "owner": "13910030347",
                    "modifier": "",
                    "path": "00019700101000000001/1B11chpTJ14000019700101000000043",
                    "shareType": 0,
                    "softLink": null,
                    "extProp1": null,
                    "extProp2": null,
                    "extProp3": null,
                    "extProp4": null,
                    "extProp5": null,
                    "ETagOprType": 1
                },
                {
                    "catalogID": "1B11chpTJ14000019700101000000044",
                    "catalogName": "手机视频",
                    "catalogType": 7,
                    "createTime": "20170329122757",
                    "updateTime": "20170329122757",
                    "isShared": false,
                    "catalogLevel": 2,
                    "shareDoneeCount": 0,
                    "openType": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "dirEtag": 4194304,
                    "tombstoned": 0,
                    "proxyID": null,
                    "moved": 0,
                    "isFixedDir": 1,
                    "isSynced": null,
                    "owner": "13910030347",
                    "modifier": "",
                    "path": "00019700101000000001/1B11chpTJ14000019700101000000044",
                    "shareType": 0,
                    "softLink": null,
                    "extProp1": null,
                    "extProp2": null,
                    "extProp3": null,
                    "extProp4": null,
                    "extProp5": null,
                    "ETagOprType": 1
                },
                {
                    "catalogID": "1B11chpTJ14018120221011123242htx",
                    "catalogName": "139邮箱",
                    "catalogType": 0,
                    "createTime": "20221011123242",
                    "updateTime": "20230830160648",
                    "isShared": false,
                    "catalogLevel": 2,
                    "shareDoneeCount": 0,
                    "openType": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "dirEtag": 4194304,
                    "tombstoned": 0,
                    "proxyID": null,
                    "moved": 0,
                    "isFixedDir": 0,
                    "isSynced": null,
                    "owner": "13910030347",
                    "modifier": "13910030347",
                    "path": "00019700101000000001/1B11chpTJ14018120221011123242htx",
                    "shareType": 0,
                    "softLink": null,
                    "extProp1": null,
                    "extProp2": null,
                    "extProp3": "{\"creator\":\"\"}",
                    "extProp4": null,
                    "extProp5": null,
                    "ETagOprType": 1
                }
            ],
            "contentList": [
                {
                    "contentID": "1B11chpTJ14001820200205235240ogr",
                    "contentName": "应急管理部机关单位内部参考手册(电子版).docx",
                    "contentSuffix": "docx",
                    "contentSize": 26761,
                    "contentDesc": "",
                    "contentType": 5,
                    "contentOrigin": 9,
                    "updateTime": "20200205235240",
                    "commentCount": 0,
                    "thumbnailURL": "",
                    "bigthumbnailURL": "",
                    "presentLURL": "",
                    "presentHURL": "",
                    "contentTAGList": null,
                    "shareDoneeCount": 0,
                    "safestate": 0,
                    "transferstate": 0,
                    "isFocusContent": 0,
                    "updateShareTime": null,
                    "uploadTime": "20200205235240",
                    "openType": 0,
                    "auditResult": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "channel": "10000000",
                    "geoLocFlag": "0",
                    "digest": "9DEB0B5FAE0D265C2E988EDDEE70EC7E",
                    "version": "",
                    "fileEtag": "1336484603161299090",
                    "fileVersion": "4194304",
                    "tombstoned": 0,
                    "proxyID": "",
                    "moved": 0,
                    "midthumbnailURL": "",
                    "owner": "13910030347",
                    "modifier": "13910030347",
                    "shareType": 0,
                    "extInfo": null,
                    "exif": {
                        "createTime": "19700101000000",
                        "longitude": null,
                        "latitude": null,
                        "localSaveTime": null
                    },
                    "collectionFlag": null,
                    "treeInfo": null,
                    "isShared": false,
                    "ETagOprType": 1
                },
                {
                    "contentID": "1B11chpTJ14001820191105115136sw6",
                    "contentName": "5G智慧云.jpg",
                    "contentSuffix": "jpg",
                    "contentSize": 551318,
                    "contentDesc": "",
                    "contentType": 1,
                    "contentOrigin": 9,
                    "updateTime": "20191105115136",
                    "commentCount": 0,
                    "presentLURL": "",
                    "presentHURL": "",
                    "contentTAGList": null,
                    "shareDoneeCount": 0,
                    "safestate": 0,
                    "transferstate": 0,
                    "isFocusContent": 0,
                    "updateShareTime": null,
                    "uploadTime": "20191105115136",
                    "openType": 0,
                    "auditResult": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "channel": "10000000",
                    "geoLocFlag": "0",
                    "digest": "98BD552639092786C91758E6D83F08ED",
                    "version": "",
                    "fileEtag": "1302963458554546635",
                    "fileVersion": "4194304",
                    "tombstoned": 0,
                    "proxyID": "",
                    "moved": 0,
                    "midthumbnailURL": "",
                    "owner": "13910030347",
                    "modifier": "13910030347",
                    "shareType": 0,
                    "extInfo": {
                        "address": "{finished:1}"
                    },
                    "exif": {
                        "createTime": "19700101000000",
                        "longitude": null,
                        "latitude": null,
                        "localSaveTime": null
                    },
                    "collectionFlag": null,
                    "treeInfo": null,
                    "isShared": false,
                    "ETagOprType": 1
                },
                {
                    "contentID": "1B11chpTJ140018201702261126056dn",
                    "contentName": "欢迎使用和彩云.pdf",
                    "contentSuffix": "pdf",
                    "contentSize": 1648058,
                    "contentDesc": "",
                    "contentType": 5,
                    "contentOrigin": 2,
                    "updateTime": "20170226112605",
                    "commentCount": 0,
                    "thumbnailURL": "",
                    "bigthumbnailURL": "",
                    "presentLURL": "",
                    "presentHURL": "",
                    "contentTAGList": null,
                    "shareDoneeCount": 0,
                    "safestate": 1,
                    "transferstate": 0,
                    "isFocusContent": 0,
                    "updateShareTime": null,
                    "uploadTime": "20170226112605",
                    "openType": 0,
                    "auditResult": 0,
                    "parentCatalogId": "1B11chpTJ14000019700101000000001",
                    "channel": "10000000",
                    "geoLocFlag": "0",
                    "digest": "9DE40A9F434BBE8F56522F0278C9723E",
                    "version": "",
                    "fileEtag": "947092153869618017",
                    "fileVersion": "4194304",
                    "tombstoned": 0,
                    "proxyID": "",
                    "moved": 0,
                    "midthumbnailURL": "",
                    "owner": "13910030347",
                    "modifier": "13910030347",
                    "shareType": 0,
                    "extInfo": null,
                    "exif": {
                        "createTime": "19700101000000",
                        "longitude": null,
                        "latitude": null,
                        "localSaveTime": null
                    },
                    "collectionFlag": null,
                    "treeInfo": null,
                    "isShared": false,
                    "ETagOprType": 1
                }
            ],
            "isCompleted": 1
        }
    }
}

**Comment by Saniq** [comment]
请问个人云请求负载在哪看

**Reply by Saniq** [reply]
谷歌版本：106.0.5249.103（正式版本） （64 位）

**Reply by Jutean** [reply]
标头旁边的载荷

**Reply by Jutean** [reply]
根文件ID怎么找

**Comment by heribe** [comment]
可以下载文件，但是上传的时候显示“该项目不在xxxxx中。请确认该项目位置，然后重试”

**Reply by heribe** [reply]
是个人云

**Reply by guoyinbin** [reply]
同问，下载可以，上传不行

**Reply by zeroduhyy** [reply]
同问同问！

**Reply by xpzz521** [reply]
你好，问下如何上传文件

**Comment by wj742421292** [comment]
添加了三个云盘后, 重新登录, 前两个提示令牌失效

**Comment by heitu001** [comment]
这破盘限制太多了不建议使用

**Comment by 27810617** [comment]
运动中国移动云盘退出网页版就会出现：令牌失效

**Reply by friedrica137** [reply]
同，只要登录者网页版139云盘就一切正常，一旦关闭网页或退出登录就gg

**Reply by gbtc007** [reply]
有没有什么办法解决这个问题啊，好烦恼啊！！！

**Comment by Ohcecho** [comment]
Fn+f12，点击network，CTRL+R刷新页面，找到文件名getdisk，在标头和负载里就可以找到所需信息

**Reply by JustinLee-2020** [reply]
负载里没找到cloudID字段啊

**Reply by tsinglinrain** [reply]
个人云不需要填入cloudID

**Comment by pwililiwp** [comment]
过一段时间就会鉴权失败

**Comment by JustinLee-2020** [comment]
获取cookie的链接打不开，页面显示“系统服务不可用”

**Comment by huweitianc** [comment]
移动云盘按标准添加了，状态不work，怎么回事

**Reply by huweitianc** [reply]
应该是alist版本问题。新版本能添加139，可惜百度网盘不能在挂载终端后在win窗口中拉取上传。无奈放弃139云盘，用老版本alist了。窗口上传下载拖拉正常。

**Comment by ericuni** [comment]
家庭云的文件管理操作, 跨存储复制支持不太完善?

**Comment by joexina** [comment]
个人云不用Cloud ID！！！！！！！！大家要记住

**Reply by drw2005** [reply]
个人云，按照着教程操作，保存时提示：Request failed with status code 504
大佬这是咋回事？

之前曾经成功挂载过，后台因为重装alist了，现在重新挂载就出现这个问题。

**Comment by ypq123456789** [comment]
按照操作一步步来的，显示文件夹了，但视频播放总是失败，显示重新连接。

**Reply by ypq123456789** [reply]
但是用浏览器底下的potplayer可以打开，很神奇。

**Reply by ypq123456789** [reply]
手机nplayer挂载webdav同样不受影响，秒开秒拉，很强！

**Reply by ypq123456789** [reply]
唉，过一段时间就会出现这种情况：移动云盘网络错误，手机也打不开文件和视频，禁用了alist 移动云盘挂载就好了！移动真让人无语

**Reply by PieJEed** [reply]
同样是这个问题

**Reply by Winyer** [reply]
俺也一样

**Comment by le45tx1m** [comment]


**Reply by le45tx1m** [reply]
经测试，填这个就对了
00019700101000000001

**Reply by xingxinag** [reply]
确实我的  ID也是
00019700101000000001

**Comment by 27810617** [comment]
如何不接收这种邮件


&nbsp;


517小骗子
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月11日(星期二) 凌晨0:33
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/docs] zh/guide/drivers/139 (Discussion #40)





 
俺也一样
 
&mdash;

**Reply by Winyer** [reply]
视频播放总是加载失败有遇到的吗

**Reply by Winyer** [reply]
从其他盘复制文件进移动云盘经常出现unexpected status code: 403这个错误
而且，几乎是百分百出现

**Comment by Winyer** [comment]
从其他盘复制文件进移动云盘经常出现unexpected status code: 403这个错误
而且，几乎是百分百出现

**Reply by Winyer** [reply]
这破网盘限制能赶上百度云盘了，快逃！

**Reply by wolfit50** [reply]
+1，确实，复制文件到移动云盘必出错

**Comment by haohao232323** [comment]
可以下载可以预览，删除就出错是咋回事提示（家庭云未知异常）

**Comment by Winyer** [comment]
5月23日家庭云删除文件出现未知异常

**Comment by dujiayao** [comment]
之前都是周期性鉴权失效，但是每次手动更新还能正常用一段时间。今天更新cookie后，一直跳“Failed init storage: 鉴权失败”，不知道是什么问题。

**Comment by myseil** [comment]
更新Cookie后提示：Failed init storage: 鉴权失败 ！个人云！

**Comment by 2030150935** [comment]
Failed init storage: 鉴权失败

**Comment by jayvzh** [comment]
Failed init storage: 鉴权失败。
看了下移动云盘更新了，cookie结尾部分增加了时间戳。是不是无解了

**Reply by feisibuke250** [reply]
同样的，完全按照教程操作的，显示：Failed init storage: 鉴权失败

**Reply by aaabbbccc100** [reply]
我也是一样的，可能要更新了。

**Comment by anorl** [comment]
Failed init storage: 鉴权失败 是不是用不了啦？

**Reply by aaabbbccc100** [reply]
我也是一样，有几天了。

**Reply by kongsoneo** [reply]
sensorsdata2015jssdkcross开头的cookie可以，用老版本移动云盘PC版点回收站进入网页版云盘
hecaiyundata2021jssdkcross开头的cookie Failed init storage: 鉴权失败，新版移动云盘pc版无法点击回收站进入网页版云盘

**Reply by ourjjj** [reply]
sajssdk_2015_cross开头也不行

**Comment by WYN156** [comment]
failed get objs: failed to list objs: 个人云未知异常
中国移动云盘是否不能再用了？

**Reply by anwen-anyi** [reply]
仔细查看文档？

**Comment by 27810617** [comment]
令牌失效

**Comment by GoofyBen82** [comment]
Failed init storage: 鉴权失败,怎么破？

**Comment by DuangXT** [comment]
移动家庭云最近报【failed get objs: failed to list objs: 待查询的父节点ID不能为空】有谁知道是怎么一回事么

**Comment by ypq123456789** [comment]
移动云盘挂载正常，但webdav看了十几个视频后，突然无法播放，回到alist界面试图保存移动云盘挂载，提示服务器内部异常。今早起来，webdav正常播放，保存移动云盘挂载正常。是不是移动那边对于webdav调用次数或流量有监测，只要超过一定次数或量，就直接短暂给你封了?

**Comment by doctorgreen4721** [comment]
个人云，使用RaiDrive挂载到本地，可以进行下载查看，但上传的时候显示“该项目不在xxxxx中。请确认该项目位置，然后重试”

**Reply by ypq123456789** [reply]
用airlivedrive可以，但移动云盘限制很多，webdav用一段时间就会无法播放，不建议用了

**Comment by yubanmeiqin9048** [comment]
用alist挂载限速30M，但是用客户端不限速，不知道有没有办法解决

**Reply by htyjswf** [reply]
是的，我也想知道咋整

**Comment by goldsoft8888** [comment]
我有一个好主意，把获取Authorization的工作变成自动化的，每次由软件自动识别并获取，不必再手工去找并添加。这应该可以做到。

**Comment by BlueSkyXN** [comment]
实测账号（黄金年费）上传1TB（个人+家庭，不确定是否计算秒传）左右后被限流，类似谷歌API403，报错是个人云未知异常

暂准确无法确定限制周期和理由，但根据web页面操作提示“单日上传超限制”可猜测是24H限流 1TB

![移动云](https://i1.wp.com/telegraph.cachefly.net/file/6ca4cb9c04c01e5caac0d.png)
![移动云](https://i1.wp.com/telegraph.cachefly.net/file/f2f253b38d8593c97b431.png)

**Reply by BlueSkyXN** [reply]
试了下可以秒传的资源，在上传额度用完之后也是不可上传的

**Comment by Thank-starlight** [comment]
怎么找不着getdisk了

**Reply by hqzmss** [reply]
移动云盘更改了api，当前方式寻找catalogID失效了

**Reply by hqzmss** [reply]
同样是电信手机号，同样的容量，同样是开通了钻石会员，一个手机号有getDisk，一个手机号没有，是个谜

**Reply by linepro6** [reply]
应该是比较新的账号没有 getDisk，可以看看有没有 list。我提交了新版个人云的 API 实现并已被合并。可以在 Action 里下载到。云盘类型选择 Personal New 就可以用了。根文件夹 ID 则填对应的 fileId。

**Reply by MerceKrist** [reply]
为什么用Personal New 类型，提示用户不存在

**Comment by waveCodeSite** [comment]
请问别的运营商手机号注册的移动云盘不能挂吗？

**Reply by BlueSkyXN** [reply]
理论上没区别

**Reply by waveCodeSite** [reply]
哦哦，我一个联通手机号注册的没有getDisk，一直不知道catalogID

**Reply by Thank-starlight** [reply]
我也没有，不知道为啥

**Reply by linepro6** [reply]
应该是比较新的账号没有 getDisk，可以看看有没有 list。我提交了新版个人云的 API 实现并已被合并。可以在 Action 里下载到。云盘类型选择 Personal New 就可以用了。根文件夹 ID 则填对应的 fileId。

**Reply by wieniawski-lai** [reply]
云盘类型没有Personal New啊

**Reply by linepro6** [reply]
> 云盘类型没有Personal New啊

要用 Action 编译的版本，release 里的没有

**Comment by htyjswf** [comment]
移动云盘老是提示转码，视频不能随意拖动进度条，而且怀疑视频上传进云盘的时候，不完整上传，md5可能不一样了。即使下载下来本地播放，放到一段进度后也错误

**Reply by BlueSkyXN** [reply]
或许可以尝试用App秒传一下

**Reply by BlueSkyXN** [reply]
也可以考虑使用“下载链接”做播放，比如alist
毕竟大部分网盘的转码功能都不行
我用的网盘中，做的最好的是阿里，其次是pcloud和od
其他比如谷歌等烂透了

**Comment by lovezongzi** [comment]
alist挂载之后播放视频，要么就是一直转圈，要么就是左上角重新连接5秒之后右下角出现三角叹号，无法播放视频，是限速的问题还是服务器带宽问题还是转码问题呀

**Reply by zzp198** [reply]
https://github.com/alist-org/alist/issues/4934

**Reply by haohao232323** [reply]
看到有大佬好像修复了这个问题，但是比较复杂没看懂，希望作者能更新一下吧

**Comment by BlueSkyXN** [comment]
今天提示鉴权失败，根据我的历史评论，恰好一个月失效

**Comment by xiangyangstu** [comment]
问一下各位大佬，怎么把别个网盘的文件下到移动盘上呀，每次下载完都说没权限，然后下载的就没用了

**Reply by BlueSkyXN** [reply]
不支持家庭云，家庭云只能下不能其他操作，写的很清楚了，不至于看不见
你需要先上传到个人盘

**Comment by wieniawski-lai** [comment]
访问这个地址:https://yun.139.com/orchestration/personalCloud/catalog/v1.0/getDisk
提示
false1010000500系统服务不可用

**Comment by nao-ye** [comment]
你好，根本没有cloud ID这个参数啊？

**Reply by xingxinag** [reply]
填这个就完了都一样
00019700101000000001

**Comment by yangjin3251** [comment]
区别就是带不带挂载盘，带挂载盘的都不行，不带挂载盘的老版本正常

**Comment by MicroNeutron** [comment]
鉴权失败

**Comment by htyjswf** [comment]
你们移动云盘测速平均多少Mbps？
我30-40

**Reply by gmugu** [reply]
300Mbps-500Mbps

**Comment by ebuact** [comment]
webdav 上传不支持中文是什么问题，今天使用群晖的cloud sync webdav挂载移动云盘进行同步，同步中发现含中文名的全部失败，于是我就测试了用阿里云盘同样的操作，阿里云盘就没有问题。移动云盘没有中文的同步也没有问题，就是只要有中文就会提示上传是被，服务器错误。

**Comment by CAI5201314** [comment]
怎么感觉移动凉了呢

**Reply by ypq123456789** [reply]
移动从来也没火过哈哈，确实有一堆问题

**Comment by wyz0211zpc** [comment]
各位老铁，使用alist上传移动云盘，单个文件最大有限制么？ 我的剩4个文件死活传不上去了，650M-1.3G不等

**Reply by my208928** [reply]
不支持中文名上传，除非你改athur，上传完再改回来

**Comment by YFCT3000** [comment]
添加了文件夹ID为什么还显示18个文件夹啊，感觉没有生效

**Comment by YFCT3000** [comment]
idm设置了16个线程，但是用idm下载文件时就只有一个线程在下载，这是为什么？

**Comment by ArmyDreamers** [comment]
挂载成功了
但显示20个空文件夹

可以复制115的视频到某个文件夹
但在移动云盘app里只能在上传记录里看到视频
也没有在复制到的那个路径文件夹里。

不知道是不是因为会员到期的原因,

我发现移动云盘会员到期 也不能在es文件管理器里登陆和查看文件了。

**Reply by ArmyDreamers** [reply]
我去 找了半天 app对应的文件夹 
原来在 

alist挂载后的  我的文件夹 里

**Comment by Xtnayue** [comment]
illegal base64 data at input byte 292
这是什么问题

**Reply by xingxinag** [reply]
failed get objs: failed to list objs: 鉴权失败
又用不了

**Comment by ArmyDreamers** [comment]
距离上次挂载成功刚好一个月。

今天显示鉴权失败

**Reply by gmugu** [reply]
移动云盘token有效期一个月，失效后需要重新获取。或者升级到最新版alist，能自动刷新token

**Reply by ArmyDreamers** [reply]
请问这么设置自动刷新token?
 不是只有authorization的选项吗

**Reply by ArmyDreamers** [reply]
是在alist管理页面点刷新就行了吗？

**Reply by gmugu** [reply]
你升级到最新的alist，系统每隔7天会在后台自动帮你刷新token（也就是authorization），不用设置。
但是你升级之后还是要先按教程获取authorization，因为你原来的authorization已经过期

**Reply by ArmyDreamers** [reply]
好的 感谢

**Reply by xingxinag** [reply]
你挂载成功吗

**Reply by ArmyDreamers** [reply]
成功了啊 两个号都成功了 一个老号 一个新个人云

**Reply by xingxinag** [reply]
我的一直报错
failed get objs: failed to list objs: 鉴权失败

**Reply by xingxinag** [reply]
我重登好几次了，刷新好几次了authorization，填上去一直显示  failed get objs: failed to list objs: 鉴权失败

**Reply by ArmyDreamers** [reply]
你看看你是不是authorization复制错了 那个basic之后的后面两个等号==也要加上。然后新个人云还是旧的要分清楚。

**Reply by xingxinag** [reply]
好我去看看

**Reply by xingxinag** [reply]
==确实没有复制上去😂😂😂😂

**Reply by xingxinag** [reply]
可是，我authorization后面与basic都没显示有==

**Reply by ArmyDreamers** [reply]
那我就不知道了

你是要挂载家庭云吗 挂个人云就够了吧 
什么cloudid 我没挂家庭云。不清楚

**Reply by xingxinag** [reply]
1、是这样basic空格232323221...........==
2、还是basic空格==21213212131321332......
？？？？？？？？？是1、还是2、

**Reply by xingxinag** [reply]
我挂的是个人

**Reply by ArmyDreamers** [reply]
那你就别管cloudid那一揽啊 你先弄清楚你是旧的还是新的个人云

反正我的旧号是这样填的 
根文件夹id填 00019700101000000001
类型 选 个人云

新号 新个人云
根文件夹id 选的 root
类型选 新的个人盘

之后就成功了。
当然对应的authorization要对。

**Reply by ArmyDreamers** [reply]
要把basic这个单词去掉啊后面也没有空格 再最后的最后有==

**Reply by xingxinag** [reply]
找到原因了，应该是新版本的alist有BUG。
我用旧版本的alist直接什么都不用弄直接填个
authorization
和id：00019700101000000001
就行了
而且authorization的那个basic值.........................的后面也不用单独加两个等号==（直接用它原始的值，啥也不用加）

**Reply by ArmyDreamers** [reply]
又是一个月鉴权失败咯。。

**Comment by BlueSkyXN** [comment]
似乎不支持上传了?

**Comment by haohao232323** [comment]


**Reply by haohao232323** [reply]
![image](https://github.com/alist-org/docs/assets/42701714/bb25f7e1-6636-48ac-a5e9-ab6573952f28)

**Comment by gxbrave** [comment]
这个其实也可以跟115一样，去手机抓包长效的Authorization

**Reply by ekyfig** [reply]
请问能不能详细一点说怎么抓

**Reply by ekyfig** [reply]
不用了，用reqable一抓就抓到了

**Reply by gdtv** [reply]
> 不用了，用reqable一抓就抓到了

请问用reqable，手机需要root吗？
@Ekyfig

**Reply by JJJJCG** [reply]
> 不用了，用reqable一抓就抓到了

老哥能详细说说吗

**Comment by zimq** [comment]
根文件夹云文件更新了，alist不更新

**Comment by gdtv** [comment]
alist上传文件提示“e.response is undefined”，复制文件提示“个人云未知异常”。后来云移动云网页版上传文件发现是“单日上传超限制”

**Comment by ekyfig** [comment]
我看好像有不用root的选项，你可以试试

________________________________

**Comment by ekyfig** [comment]
就是按照reqable的引导开始抓包，打开一次移动云盘App，然后在抓到的包列表里搜索disk，接着就跟上面的教程一样，找到请求头里的authorization

________________________________

**Comment by Kurumisann** [comment]
您好，我现在发现我在alist里进行的上传，删除操作，在网盘app和网页上都看不到了，感觉像是被分开了，请问大家有出现这个情况吗

**Reply by ekyfig** [reply]
我移动云盘也是

**Reply by gdtv** [reply]
试了一下，我这里没有这个问题。

**Reply by suixing8** [reply]
和你一样的问题，好像alist和官方渠道是两个世界似得

**Reply by suixing8** [reply]
改新个人云盘就好了

**Comment by caibirdcch** [comment]
好卡，一直转圈，有解决的办法吗？

**Comment by suixing8** [comment]
新个人云盘的目前我这边测试没有currentCatalogID，根目录文件ID节使用routerTitleShowList就可以

**Comment by vpei** [comment]
视频不能在线播放，视频文件能下载，这个是什么情况？

**Reply by vpei** [reply]
小的视频文件能播放，1.5G的电影文件就不能播放？

**Comment by heitu001** [comment]
换个网络试试



发自我的iPhone


------------------ Original ------------------
From: web ***@***.***&gt;
Date: Tue,Oct 8,2024 5:04 PM
To: alist-org/docs ***@***.***&gt;
Cc: heitu001 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [alist-org/docs] zh/guide/drivers/139 (Discussion #40)





 
视频不能在线播放，视频文件能下载，这个是什么情况？

**Reply by vpei** [reply]
感觉这个和网络没有啥关系，后来用Nova Video Player播放器中的webdav来播放，可以正常播放，只是电信的宽带播放移动的网盘视频，只有500KB左右，比较卡，基本没办法播放电影文件。

**Comment by qinlinyun** [comment]
怎么不能在线观看视频啊

**Comment by wggglggg** [comment]
不知道为什么上传大文件就提示失败，在移动云盘自家网页中上传还能秒传，试了一个2G左右的，还有5G左右的单压缩文件，好可惜，唉
